### PR TITLE
Add 11ty application

### PR DESCRIPTION
**Pull request by hamishtaplin**
Screenshot 2021-02-11 at 17 43 34

### What

Added basic 11ty/tailwindcss application:

- moved everything into `src` directory
- added 11ty app with a few example pages and nav structure
- added basic tailwindcss setup via postcss build
- added `npm-run-all` to make running multiple npm scripts easier

### How to review

- run `npm run dev` and check that the 11ty app runs successfully

### Who can review

Anyone, preferably @dandwal

### CODE REVIEW

File: src/tailwind.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+h1 {
```
Comment (hamishtaplin) [inline src/tailwind.css:5]: Ignore that these selectors are a bad idea—I just added some basic styles to page elements to get up and running and will do something more encapsulated in future.

**Review by armstrongb — COMMENTED** [review]
`package.json` line 20/25 `"license": "Open Government License v3.0",` should be `OGL-UK-3.0` as it's complaining:

>warning package.json: License should be a valid SPDX license expression

**Comment by hamishtaplin** [comment]
> `package.json` line 20/25 `"license": "Open Government License v3.0",` should be `OGL-UK-3.0` as it's complaining:
> 
> > warning package.json: License should be a valid SPDX license expression

That's not a change that was introduced in this PR

**Review by rav-pradhan — COMMENTED** [review]
As a starter to getting 11ty set up, it looks fine

Can see the build is failing in Concourse though. Was that because of Node version? It may be worth pushing an empty commit and re-triggering the build process if that has been fixed (can see other PRs have had checks pass)

**Comment by hamishtaplin** [comment]
> As a starter to getting 11ty set up, it looks fine
> 
> Can see the build is failing in Concourse though. Was that because of Node version? It may be worth pushing an empty commit and re-triggering the build process if that has been fixed (can see other PRs have had checks pass)

Yep, sorting that now.

**Review by rav-pradhan — APPROVED** [review]

**Pull request merged**